src/main/scala/com/activegrid/neo4j/AppSettingsNeo4jWrapper.scala

```diff
@@ -0,0 +1,120 @@
+        util.setNodeProperties(generalSettings, entity.settings)
+        util.setNodeProperties(authSettings, entity.authSettings)
+        val appSettings = createNode(lables.get("AS").toString)(neo)
+        appSettings --> lables.get("HGS").toString --> generalSettings
+        appSettings --> lables.get("HAS").toString --> authSettings
+
+      }
+    }
+
+  }
+
+  override def fromGraph(nodeId: Long): AppSettings = {
+    withTx {
+      neo => {
+        var genaralSettings = Map.empty[String, String];
+        var authSettings = Map.empty[String, String]
+        withTx { neo =>
+          val settingNodes = getAllNodesWithLabel(lables.get("AS").toString)(neo)
+          for (n <- settingNodes) {
+            logger.info(n.getAllProperties.toString)
+            val iterator = n.getRelationships().iterator()
+            while (iterator.hasNext) {
+              val relation: Relationship = iterator.next
+              logger.info("Processing " + relation.getType.name())
+              if (relation.getType.name.equalsIgnoreCase(lables.get("HAS").toString)) {
+                logger.info("Processing " + relation.getType.name())
+                authSettings = relation.getEndNode.getAllProperties.asScala.mapValues(_.asInstanceOf[String]).toMap
+              }
+              if (relation.getType.name.equalsIgnoreCase(lables.get("HGS").toString)) {
+                logger.info("Processing " + relation.getType.name())
+                genaralSettings = relation.getEndNode.getAllProperties.asScala.mapValues(_.asInstanceOf[String]).toMap
+              }
+            }
+          }
+          AppSettings(genaralSettings, authSettings);
+        }
+      }
+    }
+  }
+
+  def updateSettings(settingsMap: Map[String, String], relationName: String): String = {
+    withTx {
+      neo => {
+        var genaralSettings = Map.empty[String, String];
+        withTx { neo =>
+          val settingNodes = getAllNodesWithLabel(lables.get("AS").toString)(neo)
+          for (n <- settingNodes) {
+            logger.info(n.getAllProperties.toString)
+            val iterator = n.getRelationships().iterator()
+            while (iterator.hasNext) {
+              val relation: Relationship = iterator.next
+              logger.info("Processing " + relation.getType.name())
+              if (relation.getType.name.equalsIgnoreCase(relationName)) {
+                val node = relation.getEndNode;
+                settingsMap.foreach {
+                  case (k, v) => node.setProperty(k, v.toString)
+                }
+              }
+            }
+          }
+        }
+        "success"
+      }
+    }
+  }
+
+  def deleteSetting(settingsMap: Map[String, String], relationName: String): String = {
+    withTx {
+      neo => {
+        var genaralSettings = Map.empty[String, String];
+        withTx { neo =>
+          val settingNodes = getAllNodesWithLabel(lables.get("AS").toString)(neo)
+          for (n <- settingNodes) {
```

Comment:
String  "AS" is key to the value "AppSettings" label defined in the "labels" which is map.